runtime/j9vm/javanextvmi.cpp

```diff
@@ -390,9 +398,18 @@ JVM_VirtualThreadUnmountBegin(JNIEnv *env, jobject thread, jboolean lastUnmount)
 			j9object_t threadPrev = J9OBJECT_OBJECT_LOAD(currentThread, threadObj, vm->virtualThreadLinkPreviousOffset);
 			j9object_t threadNext = J9OBJECT_OBJECT_LOAD(currentThread, threadObj, vm->virtualThreadLinkNextOffset);
 
-			/* Remove thread from list. The root will never be removed. */
+			/* Non-null previous and next elements in the list suggest that the thread has
+			 * been added to the list. Only remove from the list if the previous and next
+			 * elements in the list are non-null.
+			 */
+			assert((NULL != threadPrev) && (NULL != threadNext));
```

Comment:
use `Assert_SC_true` instead

Reply:
Changed all instances of `assert` to `Assert_SC_true` in `javanextvmi.cpp`.